tests/pixie/tests/test-stdlib.pxi

```diff
@@ -333,9 +333,8 @@
 
 (t/deftest test-range
   (t/assert= (= (-seq (range 10))
-                (-seq (-iterator (range 10))
-                (reduce conj nil (range 10))
```

Comment:
Having fixed the missing bracket, this test fails because reducing on a list build a list in reverse `(9 8 7 ...)`. So I've removed it. My bad.